pubsub/google/cloud/pubsub_v1/publisher/batch/thread.py

```diff
@@ -220,9 +235,14 @@ def _commit(self):
                 self._status = base.BatchStatus.ERROR
                 exception = exceptions.PublishError(
                     'Some messages were not successfully published.')
+
                 for future in self._futures:
                     future.set_exception(exception)
 
+                _LOGGER.error(
```

Comment:
So this is _not_ bubbling up? Maybe `warnings.warn` here?

Reply:
"Bubbling up" is ambiguous: errors in this can only "bubble" up to the main thread via the future. However, we were not *logging* errors at all. With this, even if no future is used as long as logging is enabled these errors will show up *somewhere*.